x-pack/test/functional/apps/security/doc_level_security_roles.js

```diff
@@ -35,10 +35,10 @@ export default function ({ getService, getPageObjects }) {
     it('should add new role myroleEast', async function () {
       await PageObjects.security.addRole('myroleEast', {
         elasticsearch: {
-          "indices": [{
-            "names": ["dlstest"],
-            "privileges": ["read", "view_index_metadata"],
-            "query": "{\"match\": {\"region\": \"EAST\"}}"
+          'indices': [{
+            'names': ['dlstest'],
+            'privileges': ['read', 'view_index_metadata'],
+            'query': '{"match": {"region": "EAST"}}'
```

Comment:
this looks fine to me, wanted to call out on the PR because this is the only case where we had nesting of different styles. 👍